apps/pushes/views/diff.py

```diff
@@ -278,6 +232,15 @@ def content(self, path, rev):
         content = self._universal_newlines(content)
         return content
 
+    def parse(self, parser, content, data):
+        parser.readContents(content)
+        for entity in parser:
+            data[entity.key] = entity.val
+            if isinstance(entity, FluentEntity):
+                for fluent_attr in entity.attributes:
+                    data[u'{}\0{}'.format(entity.key, fluent_attr.key)] = \
```

Comment:
Is the `\0` trick really necessary? Do I understand correctly that `data` is per file? If so, the period `.` is not legal as part of the identifier in Fluent files.

Reply:
We talked about this, and no, I'll remove the stunt.